### PR TITLE
Rebuild libarchive 3.6.2 with libxml2 2.10

### PR DESCRIPTION
**Pull request by skupr-anaconda**

### CODE REVIEW

File: recipe/conda_build_config.yaml
Comment (M-Waszkiewicz-Anaconda) [inline recipe/conda_build_config.yaml:1]: I suppose those line  make the build with libxml2 v2.10  ?
why not just add in host section libxml2 2.10  ?

Comment (skupr-anaconda) [reply recipe/conda_build_config.yaml:1]: Because it's inside the general conda_build_config.yaml with another pinning

**Review by katietz — APPROVED** [review]
lgtm, thanks

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
lgtm

File: recipe/conda_build_config.yaml
Comment (akabanovs) [reply recipe/conda_build_config.yaml:1]: But if we explicitly specify the version in the host section, then probably no need for recipe's cbc?

Comment (skupr-anaconda) [reply recipe/conda_build_config.yaml:1]: Thanks. Fixed it

**Review by akabanovs — APPROVED** [review]
LGTM, thanks

**Pull request merged**